lib/Database.ts

```diff
@@ -24,6 +24,12 @@ CREATE TABLE IF NOT EXISTS Keys (
   key_hash   TEXT     NOT NULL UNIQUE,
   created_at DATETIME DEFAULT CURRENT_TIMESTAMP
 );
+CREATE TABLE IF NOT EXISTS Chat_History(
```

Comment:
Add a space where the Keys expression ends and then align the words of the schema. Basically do this: 

```sql
CREATE TABLE IF NOT EXISTS Keys (
 -- ....
); -- new line after this statement

CREATE TABLE IF NOT EXISTS Chat_History (
  id      INTEGER  PRIMARY KEY AUTOINCREMENT,
  is_user BOOLEAN  NOT NULL,
  content TEXT     NOT NULL,
  time    DATETIME DEFAULT CURRENT_TIMESTAMP
);
```

Make sure there's a single space after "Chat_History" and "(", that way the formatting is consistent.